app/models/admin/site.rb

```diff
@@ -0,0 +1,50 @@
+require 'list_processor'
+
+class Admin::Site < ActiveRecord::Base
```

Comment:
I think on reflection this would be better named as `Admin::Settings` - also use nested context do define the model, e.g.

``` ruby
module Admin
  class Settings < ActiveRecord::Base
    # implementation
  end
end
```